website/docs/r/one_dashboard_json.html.markdown

```diff
@@ -107,6 +107,400 @@ resource "newrelic_one_dashboard_json" "bar" {
   "variables": []
 }
 ```
+### Dashboard With Multiple Pages
+
+The following example demonstrates creating a dashboard with multiple pages, with each page comprising a couple of widgets.
+```hcl
+resource "newrelic_one_dashboard_json" "foo" {
+   json = file("dashboard.json")
```

Comment:
Can we use templatefile and replace the account_id variables?

Reply:
Changed this in the latest commit https://github.com/newrelic/terraform-provider-newrelic/pull/2347/commits/c290a00a888af4080aadd108f8fcd3e6cfe9d0e6.

`account_id` is now passed from the Terraform configuration into the template file.